### PR TITLE
feat: Port influxd inspect verify-tombstone subcommand

### LINKED ISSUE #19538: Port influxd inspect verify-tombstone command
## What

Port the `influxd inspect verify-tombstone` command from InfluxDB 1.x.

## How

Original 2.0 sub command:

None available.

Original 1.x source:

https://github.com/influxdata/influxdb/blob/42873d44244c23317b274221ffad8a065ae0bb0c/cmd/influx_inspect/verify/tombstone/verify.go#L31

### PR DESCRIPTION
**Pull request by DStrand1**
Closes #19538

This PR ports the `./influxd inspect verify-tombstone` subcommand. Similar to other ported commands, the logic of the verification itself is largely unchanged, with the majority of the changes in porting to Cobra and writing test cases. I'd like to point out at this point that the global structs and functions in the `cmd/influxd/inspect` package are starting to collide, so we should maybe break these commands into their own packages.

- [x] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [X] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass

### CODE REVIEW

File: cmd/influxd/inspect/verify_tombstone_test.go [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+}
+
+func WriteBadData(t *testing.T, file *os.File) {
+	t.Helper()
+
+	writer, err := os.OpenFile(file.Name(), os.O_RDWR, 0)
+	require.NoError(t, err)
+	defer writer.Close()
+
+	written, err := writer.WriteAt([]byte("foobar"), 4)
```
Comment (lesam) [inline cmd/influxd/inspect/verify_tombstone_test.go:152]: Better to just `append` I think:

https://golang.org/pkg/os/#OpenFile

```
	// Open append and write only
	f, err := os.OpenFile("access.log", os.O_APPEND|os.O_WRONLY, 0644)
```

File: cmd/influx_inspect/verify/tombstone/verify.go [hunk 2]
```diff
@@ -1,142 +0,0 @@
...
-)
-
-// Command represents the program execution for "influx_inspect verify-tombstone".
-type Command struct {
-	Stderr io.Writer
-	Stdout io.Writer
-}
-
-// NewCommand returns a new instance of Command.
-func NewCommand() *Command {
```
Comment (lesam) [inline cmd/influx_inspect/verify/tombstone/verify.go:23]: This was all dead code? That's unfortunate.

Comment (DStrand1) [reply cmd/influx_inspect/verify/tombstone/verify.go:23]: I was able to repurpose much of it, but yeah it is dead code on `master` right now.

File: cmd/influxd/inspect/verify_tombstone.go [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+	verbosity int
+	files     []string
+	f         string
+}
+
+const (
+	quiet = iota
+	verbose
+	veryVerbose
+	veryVeryVerbose
```
Comment (danxmoran) [inline cmd/influxd/inspect/verify_tombstone.go:31]: I don't see this being used anywhere, it seems like `veryVerbose` is the most verbose logging we have now.

**Review by danxmoran — APPROVED** [review]

**Pull request merged**